src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -300,20 +300,7 @@ public function initialize(array $query = array(), array $request = array(), arr
      */
     public static function createFromGlobals()
     {
-        // With the php's bug #66606, the php's built-in web server
```

Comment:
Fixed in https://github.com/php/php-src/pull/886 and part of PHP >= 5.6.13: http://www.php.net/ChangeLog-5.php#5.6.13